### DISCUSSION TITLE
Detected multiple renderers concurrently rendering the same context provider.

### DISCUSSION THREAD

**Discussion by DiegoCantu** [vercel/next.js#56901] [App Router]
This happen just after update 13.5.4 to 13.5.5

Warning: Detected multiple renderers concurrently rendering the same context provider. This is currently unsupported.
    at Provider (webpack-internal:///(ssr)/./node_modules/react-redux/lib/components/Provider.js:19:3)
    
 If I back to previous versions this warning not happen,
 
 I'm not sure if is a bug or just a warning or this still happen in 13.5.4 but now appear the warning 
 in the new version because now was detect

**Comment by kirk-admin** [comment]
I had the save problem on NextJS 14 , I manage to solve it using the latest redux versions but it requires a @types/react version, hope this helps.

"react-redux": "^9.1.1",
"redux-thunk": "^3.1.0",
"@reduxjs/toolkit": "^2.2.3",
"@types/react": "^18.2.79",
